openhcl/virt_mshv_vtl/src/processor/mshv/x64.rs

```diff
@@ -1372,7 +1370,7 @@ impl<T: CpuIo> EmulatorSupport for UhEmulationState<'_, '_, T, HypervisorBackedX
                 .vp
                 .partition
                 .hcl
-                .check_vtl_access(gpa, GuestVtl::Vtl0, flags)
+                .check_vtl_access(gpa, self.vtl, flags)
```

Comment:
This one is equivalent, given the if above, but it makes the code slightly more portable